packages/api/src/configuration/index.ts

```diff
@@ -0,0 +1,3 @@
+export const DATA_BASE_CONFIGURATION = {
```

Comment:
Use nest/env with joi validation

Reply:
As of now we can ignore the nest/env. now that is only DB config. I'll add it shortly.